app/styles/_variables.scss

```diff
@@ -40,12 +40,15 @@ $color-blue: #204762;
 
 $color-pure-blue: #00f;
 
+$color-text-blue-light: #adcce6;
 $color-gray-blue: #548288;
 $color-gray-blue-dark: #365062;
-$color-text-blue-light: #adcce6;
-$color-link-dark: #2d6a9f;
```

Comment:
This is darker than the color that replaces it, and this probably exists for a11y explicitly. Did you verify that it still passes?